### ISSUE TITLE
double click subject line, copy => copies subject + profile

### ISSUE THREAD

**Issue by jweinstein** [nylas/N1#1182]
![image](https://cloud.githubusercontent.com/assets/547354/12726236/5afba0a6-c8cb-11e5-992d-d1cd657295e2.png)

Pastes:

> this is a subject
> Inbox
> Print Thread
> 
> 10:04 AM
> 
> JeffTo: You
> 
> Write a reply…
> GITHUB
> 
> No Matching Profile

Should only copy the highlighted portion

**Comment by emorikawa** [comment]
Yes it should! Good catch. Will have to figure out a better way to isolate that while keeping the subject variable width.

**Comment by bengotow** [comment]
Maybe a seamless iframe is actually the way to go after all...

**Comment by jweinstein** [comment]
Still an issue.  Do you think you'll fix?

**Comment by emorikawa** [comment]
Yeah, let me bump this
